### ISSUE TITLE
Usar SMTP

### ISSUE THREAD

**Issue by mx-psi** [libreim/overflow#13]
**Disclaimer:** _No tengo ni idea de la mayoría de lo que estoy hablando en este issue._

Q2A da la posibilidad de gestionar las notificaciones de correo utilizando SMTP. Podríamos **utilizar una cuenta de correo externa** (source: @M42) para enviar correos de notificaciones y ~~confirmación de cuentas~~ (ya las confirma Github).

Discutir la viabilidad de esto y hacerlo en caso de que se pueda.

**Comment by robertohueso** [comment]
Me parece una buena idea para notificaciones pero para confirmación de cuentas, si usamos la cuenta de Github para registrarnos ¿No se supone que ya está verificado el correo?

**Comment by mx-psi** [comment]
Sí, no lo había pensado, actualizo eso. De todas formas con la configuración actual no debería pedir confirmación del correo.
